src/worker-helpers.js

```diff
@@ -119,7 +120,13 @@ export function getArgv(type, config, filePath, fileDir, givenConfigPath) {
   if (config.eslintRulesDir) {
     let rulesDir = resolveEnv(config.eslintRulesDir)
     if (!Path.isAbsolute(rulesDir)) {
-      rulesDir = findCached(fileDir, rulesDir)
+      const splitPath = rulesDir.split('/')
+      rulesDir = splitPath[0] === '~'
+        ? untildify(rulesDir)
```

Comment:
Does `untildify` perform any validations to check if the result actually exists?  If not, we may want to still pass the result to `findCached`.